src/olympia/addons/admin.py

```diff
@@ -616,7 +616,7 @@ def _send_mail(self, obj, action):
         send_mail(
             f'Regional Restriction {action} for Add-on',
             message,
-            recipient_list=('amo-admins@mozilla.com',),
+            recipient_list=('amo-notifications+regionrestrict@mozilla.com',),
```

Comment:
we should probably define these email addresses in constants rather than inline, but it's not new so 🤷